scripts/android.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+from upload import upload_task
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--force', dest='force', action="store_true", default=False)
+args = parser.parse_args()
+
+tasks = [
+    ("Android Project", "different parts of the Application in detail.", """Write about the basic functionality of the app, how the app should be used. Covering things narrowing to a stream or narrowing to a topic is a must include."""),
+    ("Chat Box", "chatBox (functioning and what each button does) with screenshots.", """Include text about the behavior details about swipe to remove and it automatically goes away when the main recyclerView is scrolled.\nSwitching between private and stream mode. """),
+    ("Streams Drawer", "the streams drawer (with screenshots)", """Unread counts (Currently there is a bug which is breaking the unread counts, but I hope it would be fixed soon) and it expands and shows the topics of that stream. On clicking the Stream name you can directly narrow to that stream. """),
+    ("some miscellaneous features", "some features.", """Day/Night mode\nFiltering messages\nMore """),
+    ("widget", " the usage with screenshots of the widget.", """Update time of receiving messages.\nNarrowed view in the widget."""),
+    ("menu of long click on message", " the options available when long press the message.\nWith a screenshot of the menu.", """Include documentation about each of the options."""),
+    ("supported backends", "the implemented backends in the app.\n You can get the information about these API’s at https://zulip.readthedocs.io/ ", """Include text about the 3 Authentication login backend’s the project handles currently which are Email, Google, DevAuth Backend’s."""),
+    ("API routes used in the application (for developers)", "the API routes by the Application.\nYou can get the information about these API’s at https://zulip.readthedocs.io/", """All the API’s used like get_auth_backends and information about the API’s."""),
+]
+
+description = """A descriptive text/Documentation and compiled like https://zulip.readthedocs.io/ describing """ + "%s**\n\n **Checklist**\n %s"
```

Comment:
(and then the detailed instructions, etc. go in the .md file. Check out user-guides.md for an example.)